cns/restserver/restserver.go

```diff
@@ -119,13 +119,14 @@ type EndpointInfo struct {
 }
 
 type IPInfo struct {
-	IPv4          []net.IPNet
-	IPv6          []net.IPNet `json:",omitempty"`
-	HnsEndpointID string      `json:",omitempty"`
-	HnsNetworkID  string      `json:",omitempty"`
-	HostVethName  string      `json:",omitempty"`
-	MacAddress    string      `json:",omitempty"`
-	NICType       cns.NICType
+	IPv4             []net.IPNet
+	IPv6             []net.IPNet `json:",omitempty"`
+	HnsEndpointID    string      `json:",omitempty"`
+	HnsNetworkID     string      `json:",omitempty"`
+	HostVethName     string      `json:",omitempty"`
+	MacAddress       string      `json:",omitempty"`
+	NICType          cns.NICType `json:",omitempty"`
+	NetworkNameSpace string      `json:",omitempty"`
```

Comment:
To confirm, we are adding NetworkNameSpace to the cns and ip info because this information is required during linux endpoint deletion, right?

Reply:
In the new commit I used the args from containerd to update the endpoint and no longer write the nspath to the state.